src/test/keyword-mark_test.ts

```diff
@@ -62,51 +79,56 @@ describe('keyword-mark', () => {
   });
 
   it('should react to attribute changes', () => {
-    element.setAttribute('keywords', 'ba');
+    element.setAttribute('delimiter', ':');
+    element.setAttribute('keywords', 'fo:ba');
```

Comment:
can we split this into multiple tests? one for the `keyword` attribute and one for the `delimiter` attribute